### PR TITLE
Updates to OVN & README.md

### PR DESCRIPTION
**Pull request by krishvoor**
This PR adds a new metric, to understand duration for OVN to apply
network configurations

Signed-off-by: Krishna Harsha Voora <krvoora@redhat.com>

### CODE REVIEW

**Comment by krishvoor** [comment]
The lint failure is not relevant to PR, reproduced this locally

```
[krvoora@krvoora performance-dashboards]$ for t in templates/*.jsonnet; do echo "Testing template ${t}"; ./bin/jsonnetfmt --test $t; echo 'Results:' ${?}; done
Testing template templates/api-performance-overview.jsonnet
Results: 0
Testing template templates/cilium-k8s-perf.jsonnet
Results: 0
Testing template templates/etcd-on-cluster-dashboard.jsonnet
Results: 0
Testing template templates/hypershift-performance.jsonnet
Results: 0
Testing template templates/k8s-perf.jsonnet
Results: 0
Testing template templates/kube-burner.jsonnet
Results: 0
Testing template templates/ocp-performance.jsonnet
Results: 0
Testing template templates/ovn-dashboard.jsonnet
Results: 0
Testing template templates/pgbench-dashboard.jsonnet
Results: 0
Testing template templates/uperf-perf.jsonnet
Results: 0
Testing template templates/vegeta-wrapper.jsonnet
Results: 0
Testing template templates/ycsb.jsonnet
Results: 0
[krvoora@krvoora performance-dashboards]$
```

**Comment by krishvoor** [comment]
Here's how the panel will look like, 
the panel plots against 2 pods and service for a given duration 

Pic2

Pod1

Service

**Comment by krishvoor** [comment]
Ptal: @jtaleric @chaitanyaenr @mukrishn

**Review by jtaleric — APPROVED** [review]
lgtm

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Comment by jtaleric** [comment]
@krishvoor seems to be some linting issues? I tried to see what the CI is complaining about, but I couldn't pinpoint it... Maybe run locally to determine the lint issue.

**Comment by krishvoor** [comment]
@jtaleric yes, could not pin-point CI-lint issue, however, ran the lint locally and see no issues, here are my observations,
https://github.com/cloud-bulldozer/performance-dashboards/pull/52#issuecomment-1221963276 

Any way to tell the CI to re-run the failed job?

**Review by mukrishn — APPROVED** [review]
LGTM

**Comment by krishvoor** [comment]
@jtaleric @mukrishn @chaitanyaenr please let me know if you need anything on this.

File: templates/ovn-dashboard.jsonnet [hunk 1]
```diff
@@ -180,6 +180,18 @@ local ovnCNIDel = genericGraphLegendPanel('CNI Request DEL Latency', 's').addTar
   )
 );
 
+local ovnLatencyCalculate = genericGraphLegendPanel('Duration for OVN to apply network configuration','s').addTarget(
+  prometheus.target(
+    'histogram_quantile(0.99, sum(rate(ovnkube_master_network_programming_duration_seconds_bucket[1m])) by (pod, le))',
```
Comment (rsevilla87) [inline templates/ovn-dashboard.jsonnet:185]: You should a period of 2m in rate, rather than 1m

Comment (krishvoor) [reply templates/ovn-dashboard.jsonnet:185]: Curious question, why 2m in rate?

Comment (rsevilla87) [reply templates/ovn-dashboard.jsonnet:185]: Some context -> https://www.robustperception.io/what-range-should-i-use-with-rate/

File: templates/ovn-dashboard.jsonnet [hunk 2]
```diff
@@ -180,6 +180,18 @@ local ovnCNIDel = genericGraphLegendPanel('CNI Request DEL Latency', 's').addTar
...
+    legendFormat='{{pod}} - Kind Pod',
+  )
+).addTarget(
+  prometheus.target(
+    'histogram_quantile(0.99, sum(rate(ovnkube_master_network_programming_duration_seconds_bucket[1m])) by (service, le))',
```
[the remark this answers is not in the document]
Comment (krishvoor) [reply templates/ovn-dashboard.jsonnet]: Done

File: templates/ovn-dashboard.jsonnet (hunk 1 above)
Comment (krishvoor) [reply templates/ovn-dashboard.jsonnet]: Done, updated!

**Review by rsevilla87 — APPROVED** [review]
lgtm

**Pull request merged**